normandy/classifier/tests/test_api.py

```diff
@@ -56,11 +51,26 @@ def test_it_filters_by_locale(self, client):
         recipe_names = set(r['name'] for r in res.data['recipes'])
         assert recipe_names == {'german', 'any', 'both'}
 
+    def test_it_filters_by_locale_with_json(self, api_client):
```

Comment:
This is to test [the fix in the FetchBundle view](https://github.com/mozilla/normandy/pull/78/files/0f39707be504fe890e761ed05499cf94b3a84b2e#diff-e8a07d3ca4f0d4fd6d4ea0aadb2180ddR27) where we were pulling from `request.POST` instead of `request.data`. We want to pull from `request.data` to support different encodings, like JSON. I have it reusing the filter test because the other tests only test when nothing is sent, and would pass even if data was sent as JSON because the locale param would default to None.